apps/trustlab/src/payload/collections/Helplines.js

```diff
@@ -2,6 +2,14 @@ import BaseContentCollection from "./BaseContentCollection";
 
 const Helplines = BaseContentCollection("helplines", {
   hasTags: false,
+  labels: {
+    singular: {
+      en: "Helpline",
+    },
+    plural: {
+      en: "Helplines",
```

Comment:
I think I asked this before, doesn't payload add 's' to make plural? Do we need to specify this? I know you need it for when plural and singular forms are not so direct e.g. person and people.

Reply:
You are right on this.